### ISSUE TITLE
bug: LangChain message objects are not visualized consistently in JS

### ISSUE THREAD

**Issue by Stadly** [langfuse/langfuse#1185]
### Describe the bug

When looking at the trace detail, LangChain message objects are not visualized consistently in `Input` and `Output`:
![image](https://github.com/langfuse/langfuse/assets/7263579/3e80a5b2-8026-4a55-98b2-1c849eb57595)

The verbose output of LangChain indicates that both `Input` and `Output` are LangChain message objects:
```
[llm/start] [1:llm:ChatOpenAI] Entering LLM run with input: {
  "messages": [
    [
      {
        "lc": 1,
        "type": "constructor",
        "id": [
          "langchain_core",
          "messages",
          "HumanMessage"
        ],
        "kwargs": {
          "content": "hei",
          "additional_kwargs": {}
        }
      }
    ]
  ]
}
[llm/end] [1:llm:ChatOpenAI] [1.55s] Exiting LLM run with output: {
  "generations": [
    [
      {
        "text": "Hello! How can I assist you today?",
        "message": {
          "lc": 1,
          "type": "constructor",
          "id": [
            "langchain_core",
            "messages",
            "AIMessage"
          ],
          "kwargs": {
            "content": "Hello! How can I assist you today?",
            "additional_kwargs": {}
          }
        },
        "generationInfo": {
          "finish_reason": "stop"
        }
      }
    ]
  ],
  "llmOutput": {
    "tokenUsage": {
      "completionTokens": 9,
      "promptTokens": 8,
      "totalTokens": 17
    }
  }
}
```

I would have expected the `Output` format to be similar to `Input`, and the possibility to pretty-print it, like it is for message objects like `{ role: "user", content "..." }`

### To reproduce

```ts
const llm = new ChatOpenAI({
  azureOpenAIApiKey: AZURE_OPENAI_API_KEY,
  azureOpenAIApiInstanceName: AZURE_OPENAI_API_INSTANCE_NAME,
  azureOpenAIApiDeploymentName: AZURE_OPENAI_API_DEPLOYMENT_NAME,
  azureOpenAIApiVersion: AZURE_OPENAI_API_VERSION,
  modelName: "gpt-4",
  verbose: true,
})

const handler = new CallbackHandler({
  secretKey: LANGFUSE_SECRET_KEY,
  publicKey: LANGFUSE_PUBLIC_KEY,
  baseUrl: LANGFUSE_BASE_URL,
});

await llm.invoke("hei", { callbacks: [handler] });
```

**Comment by marcklingen** [comment]
Thanks for reporting, this should be improved in v2.13.4 of the Python SDK by parsing openai/langchain chatmessage inputs more consistently.

**Comment by Stadly** [comment]
Thanks. I'm using the JavaScript SDK, version 3.0.2.

**Comment by marcklingen** [comment]
> Thanks. I'm using the JavaScript SDK, version 3.0.2.

thanks for reporting, I was to quick when reading this the first time and missed that you're on JS, updated the tags/title accordingly

**Comment by Stadly** [comment]
Am I right that this is not an issue with visualization, but rather how the SDK formats the data sent to the Langfuse server? If so, it would be great to have it prioritized, as I assume the fix will only apply to data ingested after the fix is in place.

**Comment by Stadly** [comment]
This seems to be fixed in version 3.1.0 🥳
